<launch>

<node name="ipa_hough_detection" pkg="hough_circle" type="hough_circle_node" output="screen" >
<param name="publish_frequency" value="1000" />
  </node>

<node name="rqt_reconfigure" pkg ="rqt_reconfigure" type="rqt_reconfigure" respawn="false" output="screen"/>

<node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0.25 0 0 0 1 world base_link 100" />

<node pkg="tf" type="static_transform_publisher" name="link2_broadcaster" args="0 -0.0325 1 0 0.76312478 0 0.64625116 base_link camera_link 100" />

<node pkg="tf" type="static_transform_publisher" name="link3_broadcaster" args="-0.167 0 0 0 0 0 1 base_link lidar 100" />

<node type="rviz" name="rviz" pkg="rviz" args="-d $(find hough_circle)/rviz/rviz_config.rviz"/>

<include file="$(find realsense2_camera)/launch/rs_rgbd.launch">

  <arg name="fisheye_width"       default="640"/>
  <arg name="fisheye_height"      default="480"/>
  <arg name="enable_fisheye"      default="false"/>

  <arg name="depth_width"         default="640"/>
  <arg name="depth_height"        default="480"/>
  <arg name="enable_depth"        default="true"/>

  <arg name="infra_width"         default="640"/>
  <arg name="infra_height"        default="480"/>
  <arg name="enable_infra1"       default="false"/>
  <arg name="enable_infra2"       default="false"/>

  <arg name="color_width"         default="640"/>
  <arg name="color_height"        default="480"/>
  <arg name="enable_color"        default="true"/>

 </include>




</launch>
